views/profile.handlebars

```diff
@@ -14,8 +14,8 @@
         <div class="col-sm-12 col-lg-6 my-2">
             <div class="card">
                 <div class="card-header">
-                    {{!-- TODO: Use maps.js to render this static Google Map --}}
-                    Static Google Maps
+                    <img width="600"
+                        src="https://maps.googleapis.com/maps/api/staticmap?center={{spot.latitude}},{{spot.longtitude}}&zoom=12&size=600x400&key=AIzaSyBeNBJQ5Gza1b_PNxX1_B4CKUcz6t9mozM&markers=color:red%7Clabel:C%{{spot.latitude}},{{spot.longtitude}}">
```

Comment:
Looks good. In the future, because the API key must be exposed in the HTML like this, you may want to create a restricted API key so that it can't be abused by malicious users: https://developers.google.com/maps/documentation/javascript/get-api-key#restrict_key

Reply:
The label doesn't seem to be working for me. Change it slightly to be:
`label:%7C{{spot.latitude}},{{spot.longtitude}}`